centreon/src/Core/ServiceGroup/Infrastructure/Repository/DbReadServiceGroupRepository.php

```diff
@@ -298,6 +487,318 @@ public function findByIds(int ...$serviceGroupIds): array
         return $serviceGroups;
     }
 
+    /**
+     * @param int[] $accessGroupIds
+     *
+     * @phpstan-param non-empty-array<int> $accessGroupIds
```

Comment:
This rule is not managed by this method.
Remove this phpstan rule and simply check for the presence of an empty array in the method.